src/jit/lsra.cpp

```diff
@@ -8536,6 +8545,13 @@ void LinearScan::recordMaxSpill()
         maxSpill[TYP_FLOAT] += 1;
     }
 #endif // _TARGET_X86_
+
+    if (compiler->opts.compUseSoftFP)
+    {
+        JITDUMP("Adding a spill temp for moving target address to a register.\n");
+        maxSpill[TYP_INT] += 1;
```

Comment:
Why is this needed? Where is this spill temp being used, what is it used for, and how is it obtained?